pipeline-steps/pubcloud.groovy

```diff
@@ -75,12 +75,17 @@ def cleanup(){
         )
         withEnv(["RAX_CREDS_FILE=${pyrax_cfg}"]){
           common.venvPlaybook(
-            playbooks: ['cleanup_pubcloud.yml'],
+            playbooks: ['aio-maas-cleanup.yml',
+                        'cleanup_pubcloud.yml'],
```

Comment:
It just checks that the entities are absent, so there shouldn't be any issues if they never existed in the first place.